scripts/tests.e2e.sh

```diff
@@ -14,41 +14,14 @@ fi
 AVALANCHEGO_PATH="${1-}"
 if [[ -z "${AVALANCHEGO_PATH}" ]]; then
   echo "Missing AVALANCHEGO_PATH argument!"
-  echo "Usage: ${0} [AVALANCHEGO_PATH]" >> /dev/stderr
+  echo "Usage: ${0} [AVALANCHEGO_PATH]" >>/dev/stderr
   exit 255
 fi
 
-# Set the CGO flags to use the portable version of BLST
```

Comment:
Why are we removing this? Don't we need it to ensure this works locally on MacOS?

Reply:
`install_anr.sh` includes: `source "$AVALANCHE_PATH"/scripts/constants.sh` which already includes:
```
# Set the CGO flags to use the portable version of BLST
#
# We use "export" here instead of just setting a bash variable because we need
# to pass this flag to all child processes spawned by the shell.
export CGO_CFLAGS="-O -D__BLST_PORTABLE__"
# While CGO_ENABLED doesn't need to be explicitly set, it produces a much more
# clear error due to the default value change in go1.20.
export CGO_ENABLED=1
```